### ISSUE TITLE
Screen Reader: Many Pages: Plain text elements announced as actionable

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77496]
## Action Performed:
Precondition: User is logged in

1. Using TalkBack on Android, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Wallet button and double-tap to activate it
4. Swipe right to focus the Expensify card menu item and double-tap to activate it
5. Swipe right to focus the Virtual card number text and note how it is announced

Other Occurrences:
1. Security: all accounts under "You can access these accounts via the account switcher" text

## Expected Result:
The virtual card number should be announced as static text, not a button or actionable element.

## Actual Result:
The card number is announced as actionable with "double-tap to activate" hint, even though it is plain text.
Double-tapping triggers no action.

## Area issue was found in:
Expensify Card

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Remove any interactive role from the static texts.
https://appt.org/en/docs/react-native/samples/accessibility-role

## Environment(s)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
<img width="1080" height="2195" alt="Image" src="https://github.com/user-attachments/assets/9d94a95a-7c58-4266-a018-17f86594fa30" />

https://github.com/user-attachments/assets/a5a14089-7212-43c4-ade8-f5218e8c1203

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7008140
**Issue reported by:** NA
**Slack conversation:** NA
